gemfiles/Gemfile.activerecord-5.0.x

```diff
@@ -0,0 +1,10 @@
+source "https://rubygems.org"
+
+gemspec :path => '..'
+
+gem "activerecord", "~> 5.0.0"
+gem "pg", "~> 0.15"
+
+unless ENV['CI'] || RUBY_PLATFORM =~ /java/
+  gem 'byebug'
```

Comment:
I haven't personally been using this gem, so if you want to remove it entirely that's fine with me.